data.sql

```diff
@@ -36,3 +36,37 @@ Set owner_id = 2 WHERE name IN ('Gabumon', 'Pikachu'),
 Set owner_id = 3 WHERE name IN ('Devimon', 'Plantmon'),
 Set owner_id = 4 WHERE name IN ('Charmander', 'Squirtle', 'Blossom'),
 Set owner_id = 5 WHERE name IN ('Angemon','Boarmon');
+
+INSERT INTO vets(name, age, date_of_graduation)
+VALUES ('William Tatcher', 45, 'Apr 23rd, 2000'),
+('Maisy Smith', 26, 'Jan 17th, 2019'),
+('Stephanie Mendez',64, 'May 4th, 1981'),
+('Jack Harkness', 38, 'Jun 8th, 2008');
```

Comment:
- [ ] Good job so far 👍 Kindly use the correct date format in adding data to your table. When I tried recreating your database, this date format kept giving errors. The correct format for `Apr 23rd, 2000` is `04-23-2000`. Kindly update all your dates using this format 👍 
- Your database:
![Annotation 2022-04-28 052823](https://user-images.githubusercontent.com/85113567/165677161-19861b93-2d0e-4d82-afb9-a21202982258.jpg)